Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerFrequentConfigurationIssues.ps1

```diff
@@ -318,7 +318,8 @@ function Invoke-AnalyzerFrequentConfigurationIssues {
             $showMoreInfo = $true
         }
 
-        if ($connector.TlsDomain -ne "mail.protection.outlook.com") {
+        if ($connector.TlsDomain -ne "mail.protection.outlook.com" -and
```

Comment:
The change would work. However, I'm thinking if we should validate other clouds as well (e.g, GCC or GCC High as they use AFAIK `mail.protection.office365.us` or 21Vianet (not sure about the domain))

https://learn.microsoft.com/en-us/microsoft-365/enterprise/dns-records-for-office-365-gcc-high?view=o365-worldwide

Reply:
Would we consider those cloud types of EXO connectors even based off our current logic?

Reply:
@lusassl-msft for now, we are going to go ahead and merge this. We should create an issue on this and work in logic for those other cloud options that we do provide.

Reply:
Need to validate this. I'll file a work item to track this. For now, the change should be good.